src/main/java/TaskManager.java

```diff
@@ -0,0 +1,87 @@
+public class TaskManager {
```

Comment:
As per comments in Duke.java, consider adding whitespaces between keywords, parenthesis and braces, e.g. "if (condition) {" and putting else on the same line as the closing braces e.g. "} else {".